docs/packages/cookbook/article-tagging.rst

```diff
@@ -0,0 +1,93 @@
+Auto-Tagging and Querying Articles
```

Comment:
worth linking to the related blog post or fuller implementation anywhere as a "Learn More" or something ?